bitnami/kibana/README.md

```diff
@@ -0,0 +1,310 @@
+# Kibana
+
+[Kibana](https://kibana.com/) is an open source, browser based analytics and search dashboard for Elasticsearch.
+
+## TL;DR;
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+
+# Option 1: With an existing Elasticsearch instance
+$ helm install bitnami/kibana --set elasticsearch.external.hosts[0]=<Hostname of your ES instance> --set elasticsearch.external.port=<port of your ES instance>
+
+# Option 2: With a bundled Elasticsearch instance
+$ helm install bitnami/kibana --set elasticsearch.enabled=true
+```
+
+## Introduction
+
+This chart bootstraps a [kibana](https://github.com/bitnami/bitnami-docker-kibana) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters.
+
+## Installing the Chart
+
+This chart requires a Elasticsearch instance to work. The chart offers two options:
+
+- Use an already existing Elasticsearch instance.
+- Deploy a new Elasticsearch instance.
```

Comment:
I think it's worth mentioning that by default (`helm install bitnami/elasticsearch`) it'll deploy a new ES instance.

Reply:
+1

Reply:
I'm not happy with that, but helm lint will fail otherwise.

Reply:
Done